test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -73,13 +73,6 @@ defmodule Membrane.RTC.HLSEndpointTest do
 
       assert Enum.sort(directory_files) == reference_dir |> File.ls!() |> Enum.sort()
 
-      for file <- directory_files do
-        output_path = Path.join(output_dir, file)
-        reference_path = Path.join(reference_dir, file)
-
-        assert File.read!(output_path) == File.read!(reference_path)
-      end
```

Comment:
After internal discussion it looks like the date which makes it impossible to compare file contents should be either made mockable in hls plugin or somehow excluded from comparing in tests (e.g. by using mode :vod in tests)?